src/www/base.webpack.js

```diff
@@ -47,6 +48,14 @@ exports.makeConfig = (options) => {
       extensions: ['.tsx', '.ts', '.js'],
       fallback: {'url': require.resolve('url/')},
     },
+    optimization: {
+      minimizer: [new TerserPlugin({
+        terserOptions: {
+          keep_classnames: true,
```

Comment:
That will probably make the minimization significantly less efficient.
I wonder if we should restrict to specific types or files.

Reply:
I compared the minified file size and it only differs by 200KB with this change.